src/components/Content.js

```diff
@@ -0,0 +1,168 @@
+import { useEffect, useState } from 'react';
+import ProductsView from './ProductsView';
+import FilterView from './FilterView';
+import Cart from './Cart';
+
+
+import {
+  MDBBtn,
+  MDBModal,
+  MDBModalDialog,
+  MDBModalContent,
+  MDBModalHeader,
+  MDBModalTitle,
+  MDBModalBody,
+} from 'mdb-react-ui-kit';
+
+
+function ModalCart({showCart,setShowCart,cart,setCart}) {
+  // const [scrollableModal, setScrollableModal] = useState(true);
+  
+  return (
+      <MDBModal className="custom-modal" show={showCart} setShow={setShowCart} tabIndex='-1'>
+        <MDBModalDialog className="custom-modal-body" scrollable>
+          <MDBModalContent>
+            <MDBModalHeader>
+              <MDBModalTitle className="cart-modal-title">{"Cart"}</MDBModalTitle>
+              <MDBBtn
+                className='btn-close'
+                color='none'
+                onClick={() => {setShowCart(!showCart)}}
+              ></MDBBtn>
+            </MDBModalHeader>
+            <MDBModalBody>
+              <Cart cart={cart} setCart={setCart}/>
+            </MDBModalBody>
+          </MDBModalContent>
+        </MDBModalDialog>
+      </MDBModal>
+  );
+}
+
+function Content({showCart, setShowCart,searchString}) {
+  
+  const [products, setProducts] = useState([]);
+  const [filters, setFilters] = useState({});
+  const [cart, setCart] = useState({});
+  const [resultMessage, setResultMessage] = useState('Showing All Products');
+  const [pageNo, setPageNo] = useState(1);
+
+  const fetchProductsData = (page,filters,searchString) => {
+    var brands = filters.brand ? Array.from(filters.brand) : [];
+    var params = { brand: brands, searchString:searchString};
+    var url = '?' + ( new URLSearchParams( params ) ).toString();
```

Comment:
@rahulmeghwal I'm not sure I'm following how the array of brands would convert into the query string param. I assume it would be the "brand" as the query string param key, but how would the value be formatted?

Reply:
You are right, the key is 'brand' and the value is an array of all selected brands, this array is formatted to a comma separated array values. For example one sample query which you can run from browser is : http://3.86.87.11:8080/getProducts/1?brand=IMARA,Biba,Libas or http://3.86.87.11:8080/getProducts/1?brand=IMARA%2CBiba%2CLibas (after URL encoding)